tests/test_examples.py

```diff
@@ -46,6 +46,20 @@ def test_generation(self):
             "test token_acc = {}\nLOG:\n{}".format(test['token_acc'], stdout),
         )
 
+    @testing_utils.retry(ntries=3)
+    def test_repeater(self):
+        """
+        Test a simple TRA based bag-of-words model.
+        """
+        stdout, valid, test = testing_utils.train_model(
+            dict(task='integration_tests', model='examples/tra', eps=1, bs=100)
+        )
+        self.assertEqual(
+            test['exs'],
+            100,
+            'test examples = {}\nLOG:\n{}'.format(valid['exs'], stdout),
+        )
```

Comment:
do you think there should be more checks than just "it did 100 examples"?

Reply:
I don't know what else should be checked? Like a certain performance?

Reply:
I added the test to check accuracy for test and validation as well.